.travis.yml

```diff
@@ -1,5 +1,7 @@
+dist: trusty
```

Comment:
I heard a rumor from the summer school that psi4 and trusty don't play well together, hence pinning precise. I guess we'll see from this PR.

Commented-out sections in .travis file are just old py/compiler combinations. I think we're pretty satisfied with the time/coverage balance of 3 tests, so the others could be removed. Just leave [this commented section](https://github.com/psi4/psi4/blob/master/.travis.yml#L4-L13) as it took me a lot of searching to figure out allowed fails.

Reply:
It seems to be working on trusty.